tests/core/TransactionProcessor.spec.ts

```diff
@@ -26,14 +26,24 @@ describe('TransactionProcessor', () => {
   let downloadManager: DownloadManager;
   let blockchain: IBlockchain;
   let transactionProcessor: TransactionProcessor;
+  let versionMetadataMapper: any = {};
+  let versionMetadata: any;
```

Comment:
`const` instead of `let`.

Reply:
linter should be catching these.

Reply:
It's not because I `let` ant then assign value to it, idk why I did that haha, probably did some refactoring and ended up like this. Will change it.